### ISSUE TITLE
Utilize PIO?

### ISSUE THREAD

**Issue by BjornTheProgrammer** [tuupola/hagl_pico_mipi#21]
More of a question than an issue, but would utilizing PIO be a better strategy for communicating with the TFT screen? I notice that [this](https://github.com/zapta/pio_tft) library utilizes PIO for the pico driver. It is not better to utilize PIO as it utilizes a different processor than the cortex m0+?

I appreciate that you've made this library, absolutely amazing work!

**Comment by tuupola** [comment]
Yeah option to use PIO is in my todo list. Same thing for 8080 displays. Currently just paid work comes first so I cannot use as much time for my open source projects as I would like to :(
